starter-code/routes/index.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express');
+const router  = express.Router();
+const Celebrity = require("../models/celebrity.js");
```

Comment:
Actually there is something wrong with your code, you can't populate the database with entry that are based on the Movie model, and try to call the celebrity one for the render.

It wasn't working until I switch the code to this : 

```js

const Movie = require("../models/movie.js");

//-------------ROUTING
router.get("/celebrities", (req, res, next) => {
  Movie.find()

```